### PR TITLE
test/kubernetes/e2e: migrate helm tests

### PR DESCRIPTION
**Pull request by sam-heilbron**
# Description

## Docs changes

# Context

## Notes for reviewers

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: test/kubernetes/e2e/features/helm_settings/suite.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	"github.com/stretchr/testify/suite"
+)
+
+var _ e2e.NewSuiteFunc = NewTestingSuite
+
+// testingSuite is a Suite of tests that is used to validate that the manifests
+// used in our Helm unit tests are valid
+//
+// These tests are defined separately from our `features/helm` tests, to ensure they are run against
+// a standalone cluster, and do not impact those other tests.
```
Comment (jbohanon) [inline test/kubernetes/e2e/features/helm_settings/suite.go:25]: nit: This doesn't _ensure_ they are run against a different cluster does it? Specifying them both on the same suite runner will cause them to be run against the same cluster, just not concurrently?

Comment (sam-heilbron) [reply test/kubernetes/e2e/features/helm_settings/suite.go]: great catch, cluster was the wrong word...i meant installation

Comment (jbohanon) [reply test/kubernetes/e2e/features/helm_settings/suite.go]: I don't think it's a different installation either, is it..? https://github.com/solo-io/gloo/blob/c352fdaf2d8ddf4653b5ba660a57c13650f1de56/test/kubernetes/e2e/tests/helm_test.go#L49-L52

Comment (sam-heilbron) [reply test/kubernetes/e2e/features/helm_settings/suite.go]: oof yeah great catch. i wanted to isolate these, and misused the runner. i can update to try to ensure we install..run this test..uninstall. do you think that would be useful?

**Review by jbohanon — APPROVED** [review]

**Review by davidjumani — APPROVED** [review]

**Comment by sam-heilbron** [comment]
dozer

**Pull request merged**